tests/ha/priority_fencing_delay.pm

```diff
@@ -13,7 +13,8 @@ use warnings;
 use testapi;
 use lockapi;
 use hacluster;
-use utils qw(zypper_call);
+use utils qw(zypper_call reconnect_mgmt_console);
+use Utils::Backends 'is_pvm';
```

Comment:
Not a big deal, but better use `qw(is_pvm)` instead of `'is_pvm'`. Makes adding more functions from that library easier, and it's more uniform with the other `use` lines in this module.

Reply:
good suggestion, and updated the code.